code/src/cljs/sixsq/nuvla/ui/edge_detail/views.cljs

```diff
@@ -231,14 +231,47 @@
           [ui/Segment
            [:b "Update to: "]
            [DropdownReleases {:placeholder "select a version"
-                              :on-change   (ui-callback/value #(on-change-fn %))}]]]
+                              :on-change   (ui-callback/value #(on-change-fn %))}]]
+          [uix/Accordion
+           [:<>
+            [ui/Form
+             [ui/FormInput {:label         "Project"
+                            :placeholder   "nuvlabox"
+                            :required      true
+                            :default-value (:project-name @form-data)
+                            :on-change     (ui-callback/input-callback
+                                             #(swap! form-data assoc :project-name %))}]
+             [ui/FormInput {:label         "Working directory"
+                            :placeholder   "/home/ubuntu/nuvlabox-engine"
+                            :required      true
+                            :default-value (:working-dir @form-data)
+                            :on-change     (ui-callback/input-callback
+                                             #(swap! form-data assoc :working-dir %))}]
+             [ui/FormField
+              [:label [general-utils/mandatory-name "Config files"]]
+              [ui/TextArea {:placeholder   "docker-compose.yml\ndocker-compose.gpu.yml\n..."
+                            :required      true
+                            :default-value (:config-files @form-data)
+                            :on-change     (ui-callback/input-callback
+                                             #(swap! form-data assoc :config-files %))}]]
+             [ui/FormField
+              [:label "Environment"]
+              [ui/TextArea {:placeholder   "NUVLA_ENDPOINT=nuvla.io\nPYTHON_VERSION=3.8.5\n..."
+                            :default-value (:environment @form-data)
+                            :on-change     (ui-callback/input-callback
+                                             #(swap! form-data assoc :environment %))}]]
+             ]]
+           :label "Advanced"
```

Comment:
at least this one should be on the dictionary, and also line 233 above

Reply:
@cjdcordeiro done